File: baseLib/src/androidTest/java/me/ycdev/android/lib/common/AndroidLibTestApplication.java
package me.ycdev.android.lib.common;

import android.app.Application;

import me.ycdev.android.lib.common.utils.DebugUtils;
import me.ycdev.android.lib.common.utils.LibLogger;
import timber.log.Timber;

public class AndroidLibTestApplication extends Application {
    private static final String TAG = "BaseLibTestApp";

    @Override
    public void onCreate() {
        super.onCreate();
        LibLogger.d(TAG, "onCreate");
        DebugUtils.enableStrictMode();
        Timber.plant(new Timber.DebugTree());
    }
}

File: baseLib/src/androidTest/java/me/ycdev/android/lib/common/AndroidLibTestApplication.kt
package me.ycdev.android.lib.common

import android.app.Application

import me.ycdev.android.lib.common.utils.DebugUtils
import me.ycdev.android.lib.common.utils.LibLogger
import timber.log.Timber

class AndroidLibTestApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        LibLogger.d(TAG, "onCreate")
        DebugUtils.enableStrictMode()
        Timber.plant(Timber.DebugTree())
    }

    companion object {
        private const val TAG = "BaseLibTestApp"
    }
}
